pkg/datapath/linux/config/config.go

```diff
@@ -571,6 +571,10 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["ENABLE_ICMP_RULE"] = "1"
 	}
 
+	if option.Config.EnableL7Proxy {
+		cDefinesMap["ENABLE_L7_PROXY"] = "1"
+	}
```

Comment:
@vincentmli Any reason you added this? I don't see it used anywhere in the BPF codebase.

Reply:
I may have added it while in the middle of developing this feature and ended up not using it

Reply:
Ok. I'll send a PR to remove it.